blues/example.py

```diff
@@ -35,17 +34,17 @@ def runNCMC(platform_name):
 
     #Define the 'model' object we are perturbing here.
     # Calculate particle masses of object to be moved
-    ligand = ncmc.Model(struct, 'LIG')
+    ligand = RandomLigandRotationMove(struct, 'LIG')
     ligand.calculateProperties()
 
     # Initialize object that proposes moves.
-    ligand_mover = ncmc.MoveProposal(ligand, 'random_rotation', opt['nstepsNC'])
+    ligand_mover = MoveEngine(ligand)
```

Comment:
So if we want to do multiple move types on the ligand, how would this work? From a quick glance here, it would end up looking something like
```
ligand_rot = RandomLigandRotationMove(struct, 'LIG')
ligand_dart = SmartDartMove(struct, 'LIG') 

ligand_mover = MoveEngine([ligand_rot, ligand_dart])
```

Where I think it should end up looking is like:
```
ligand = { <resid of ligand> : parmed.Structure or atom indices, moves : [ 'random_rotation', 'smart_dart',...] } 
ligand_mover = MoveEngine( ligand ) 
```